molo/core/tests/test_admin_view.py

```diff
@@ -0,0 +1,162 @@
+from django.test import TestCase
+from django.contrib.auth.models import User
+from wagtail.wagtailimages.tests.utils import get_test_image_file, Image
+
+from molo.core.models import (
+    Main, SiteLanguageRelation, Languages,
+    ArticlePage)
+from molo.core.tests.base import MoloTestCaseMixin
+
+
+class TestAdminView(TestCase, MoloTestCaseMixin):
+    def setUp(self):
+        self.mk_main()
+        main = Main.objects.all().first()
+        self.english = SiteLanguageRelation.objects.create(
+            language_setting=Languages.for_site(main.get_site()),
+            locale='en',
+            is_active=True)
+
+        self.french = SiteLanguageRelation.objects.create(
+            language_setting=Languages.for_site(main.get_site()),
+            locale='fr',
+            is_active=True)
+
+        self.yourmind = self.mk_section(
+            self.section_index, title='Your Mind')
+
+        self.yourmind_fr = self.mk_section_translation(
+            self.yourmind, self.french, title='Your mind in french')
+
+        self.mk_main2()
+        self.main2 = Main.objects.all().last()
+        self.language_setting2 = Languages.objects.create(
+            site_id=self.main2.get_site().pk)
+        self.english2 = SiteLanguageRelation.objects.create(
+            language_setting=self.language_setting2,
+            locale='en',
+            is_active=True)
+
+        self.spanish = SiteLanguageRelation.objects.create(
+            language_setting=self.language_setting2,
+            locale='es',
+            is_active=True)
+
+        self.yourmind2 = self.mk_section(
+            self.section_index2, title='Your mind2')
+
+    def test_articles_appears_in_admin_view(self):
+        User.objects.create_superuser(
```

Comment:
@cblignaut You could move this into the `setup` function to avoid repeating it for each test as well as the login function and just refer to `self.client`